bricksrc/quantities.py

```diff
@@ -1,109 +1,565 @@
-from rdflib import Literal
-from .namespaces import SKOS, OWL, BRICK
+from brickschema.graph import Graph
+from brickschema.inference import BrickInferenceSession
+from rdflib import Literal, URIRef
+from .namespaces import SKOS, OWL, RDFS, BRICK, QUDTQK, QUDTDV, QUDT, UNIT
 
 
+g = Graph()
+g.load_file("support/VOCAB_QUDT-QUANTITY-KINDS-ALL-v2.1.ttl")
+g.load_file("support/VOCAB_QUDT-UNITS-ALL-v2.1.ttl")
+g.g.bind("qudt", QUDT)
+g.g.bind("qudtqk", QUDTQK)
+sess = BrickInferenceSession()
+g = sess.expand(g)
+
+
+def get_units(brick_quantity):
+    """
+    Fetches the QUDT unit and symbol (as a Literal) from the QUDT ontology so
+    in order to avoid having to pull the full QUDT ontology into Brick
+    """
+    res = g.query(
+        f"""SELECT ?unit ?symbol WHERE {{
+                    <{brick_quantity}> qudt:applicableUnit ?unit .
+                    ?unit qudt:symbol ?symbol .
+                    FILTER(isLiteral(?symbol))
+                    }}"""
+    )
+    for r in res:
+        yield r
+
+
+"""
+Each is a qudt:QuantityKind
+"""
 quantity_definitions = {
     "Air_Quality": {
-        "subclasses": {
-            "CO2_Level": {},
-            "PM10_Level": {},
-            "PM25_Level": {},
-            "TVOC_Level": {},
+        "subconcepts": {
+            "CO2_Level": {
+                QUDT.applicableUnit: [UNIT.PPM],
+                QUDT.hasDimensionVector: QUDTDV["A0E0L0I0M0H0T0D1"],
+                SKOS.definition: Literal("The concentration ratio of some substance"),
```

Comment:
The substance should be explicitly `CO2` in the definition.

Reply:
I'm not sure what this means, but it seems perhaps more appropriate to separate out the `CO2`/`PM2.5` aspects of the quantity into substances

Reply:
I was referring to the exact line of `SKOS.definition: Literal("The concentration ratio of some substance"),` The definitions should say "The concentration ratio of CO2" instead of some substance.

Reply:
This is already done